Sources/SafeDICore/Generators/DependencyTreeGenerator.swift

```diff
@@ -136,11 +138,7 @@ public final class DependencyTreeGenerator {
             }
         }
 
-        struct UnfulfillableProperty: Hashable, Comparable {
```

Comment:
we don't need the property to be comparable because the generated error is already comparable and we sort that